core/templates/pages/exploration-editor-page/translation-tab/services/translation-topic.service.ts

```diff
@@ -31,7 +31,10 @@ import { LoggerService } from 'services/contextual/logger.service';
   providedIn: 'root'
 })
 export class TranslationTopicService {
-  private activeTopicName: string = null;
+  // This property is initialized using Angular lifecycle hooks
```

Comment:
```suggestion
  // This property is initialized using async methods
```
Let's not use `Angular Lifecycle hooks` term in *.service.ts files.

Reply:
Done